project_purchase_line_link/models/project_project.py

```diff
@@ -0,0 +1,34 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, _, fields, api
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    purchase_order_lines_count = fields.Integer(compute='_compute_purchase_order_lines_count', groups='purchase.group_purchase_user')
+
+    @api.depends('analytic_account_id')
+    def _compute_purchase_order_lines_count(self):
+        purchase_orders_line = self.env['purchase.order.line'].search([
+            ('account_analytic_id', '=', self.analytic_account_id.ids)

```

Comment:
Wrong operator (as more than one analytic accounts could be obtained)
```suggestion
            ('account_analytic_id', 'in', self.analytic_account_id.ids)

```